Database/Updates/Character/2017_03_29_character_vw_character_positions.sql

```diff
@@ -0,0 +1,5 @@
+DROP VIEW IF EXISTS `vw_character_positions`;
```

Comment:
This script creates a database `view`, used for getting all of the positions for a single character.